_posts/2018-12-19-presentation: some thinking.markdown

```diff
@@ -0,0 +1,12 @@
+
+Sorry for the late presentation for which I would like to blame the procrastination of humanity. There are something I would like to talk about after the last 2 weeks' work, after think through.
```

Comment:
I am not quite sure what you are trying with this post.
Is this a summary or reflection on a presentation that you've saw? Or you just want to write some life reflections on the events you've gone through? I think you should clearly state the purpose of this writing and why you are sharing this.

Reply:
I have deleted that blog which it's suppose write to myself.